addons/account/controllers/onboarding.py

```diff
@@ -16,7 +16,7 @@ def account_invoice_onboarding(self):
             return {}
 
         return {
-            'html': request.env.ref('account.account_invoice_onboarding_panel')._render({
+            'html': request.env['ir.qweb']._render('account.account_invoice_onboarding_panel', {
```

Comment:
This change will be a real pain in the ass for localizations/forward-ports. Instead of all these changes, keep the `_render` method on the view as a placeholder to ther ir.qweb method.

Reply:
you can make an easy search & replace like
```
env.ref\(['"]([\w.]*)['"]\)._render\(
env['ir.qweb']._render('$1', 
```
in sublime to replace everything. The goal of this PR was to simplify the API and avoid the mixup of environment to access the report record and the environment to evaluate it. Keeping an old method would remove this.

Reply:
I don't agree with you. This is not making the code cleaner nor easier to use. Instead, this will make forward-port harder. But this is just my opinion.

Reply:
The problem of `ViewRecord._render(data)` (and the whole report API that will be the subject of future PRs) is the ambiguity between the environment to access the view/report record (private, needs a sudo) and the data to render it (often current user, except portal, etc.). We had some security issues with that and the best way to avoid this was to refactor the render API to never work with view/report records directly (see task id 2670865 for discussions of the alternatives).
A goal was also to avoid the multiplication of "render" methods (between the ones on ir.actions.report, ir.ui.view or ir.qweb) and this PR does remove one abstraction level.